ui/browser/views/navbar/location.jsx

```diff
@@ -20,15 +20,52 @@ import { fixURL, getCurrentWebView } from '../../browser-util';
  * to ipcRenderer on mount, and because the key-down event handler is non
  * trivial.
  */
-class Location extends Component {
-  constructor() {
-    super();
-    this.handleKeyDown = this.handleKeyDown.bind(this);
-  }
+const Location = createClass({
```

Comment:
Extending the React.Component class resulted in many warnings from React, it seems not all methods are supported there (getDefaultState() was the one that started it, IIRC)